orchestrator/main_loops.go

```diff
@@ -90,15 +88,13 @@ func (p *peggyOrchestrator) EthOracleMainLoop(ctx context.Context) (err error) {
 
 		lastCheckedBlock = currentBlock
 
-		/*
-			Auto re-sync to catch up the nonce. Reasons why event nonce fall behind.
-				1. It takes some time for events to be indexed on Ethereum. So if peggo queried events immediately as
-				   block produced, there is a chance the event is missed. We need to re-scan this block to ensure events
-				   are not missed due to indexing delay.
-				2. if validator was in UnBonding state, the claims broadcasted in last iteration are failed.
-				3. if infura call failed while filtering events, the peggo missed to broadcast claim events occurred in
-				   last iteration.
-		**/
+		// Auto re-sync to catch up the nonce. Reasons why event nonce fall behind.
+		//	1. It takes some time for events to be indexed on Ethereum. So if peggo queried events immediately as
+		//	   block produced, there is a chance the event is missed. We need to re-scan this block to ensure events
+		//	   are not missed due to indexing delay.
+		//	2. if validator was in UnBonding state, the claims broadcasted in last iteration are failed.
+		//	3. if infura call failed while filtering events, the peggo missed to broadcast claim events occurred in
```

Comment:
We cannot assume what provider is used for eth nodes.

```suggestion
		//	3. If the ETH call failed while filtering events, the peggo missed to broadcast claim events occurred in
```